app/assets/stylesheets/partials/_elections.scss

```diff
@@ -57,16 +42,26 @@ img {
   }
 }
 
-.btn-election {
-  background-color: transparent;
-  border-color: $black;
-  border-radius: 10px;
-  color: $black;
-  font-size: 18px;
-  padding: 10px 25px;
+.btn {
+  &.election {
+    background-color: transparent;
+    border-color: $black;
+    border-radius: 10px;
+    color: $black;
+    font-size: 18px;
+    padding: 10px 25px;
 
-  &:hover {
-    border-color: $main-color;
-    color: $main-color;
+    &:hover {
+      border-color: $main-color;
+      color: $main-color;
+    }
   }
 }
+
+.center {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  min-width: 40%;
```

Comment:
Properties should be ordered display, margin-left, margin-right, max-width, min-width